docs/source/commands/contracts.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: Rover contract commands
+description: For use with Apollo Contract variants
+---
+
+A **contract** is a variant that is the result of filtering on a source variant:
+
+Rover commands that interact with contracts begin with `rover contract`.
+
+## Describing a contract variant
+
+These commands enable you to describe a contract in a federated graph.
+
+### `contract describe`
+
+TODO(caydie) is this correct? I'm assuming you always have to authn to rover with studio 
+> This command requires [authenticating Rover with Apollo Studio](../configuring/#authenticating-with-apollo-studio).
+
+You can use Rover to describe the contract variant of any source variant that Rover has access to.
+
+Run the `contract describe` command, like so:
+
+```bash
+rover contract describe my-graph@my-contract-variant
+```
+
+The argument `my-graph@my-contract-variant` in the example above is a [graph ref](../conventions/#graph-refs) that specifies the ID of the Studio graph you're fetching from, along with which [variant](/graphos/graphs/overview/#variants) you're fetching.
+
+The output will contain information of how the contract variant was configured such as the source variant, include and exclude tags.
```

Comment:
I think that once we have an example of what this output looks like (when the command is implemented) we should include that here